notebooks/02-Summary-statistics-and-plots.ipynb

```diff
@@ -254,14 +254,48 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "9820d259-680e-49fd-a1eb-e14d200a0b7a",
+   "id": "afd797d8-9775-4f62-8da9-d135eaa34b49",
    "metadata": {},
    "outputs": [],
    "source": [
     "df_three_images[\"contour_lengths\"].groupby(\"image\").plot.hist(\n",
     "    figsize=(16, 9),\n",
     "    bins=20,\n",
+    "    title=\"Contour Lengths\",\n",
+    "    alpha=0.5,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b362e9af-8577-4c44-8afb-199ef2535607",
+   "metadata": {},
+   "source": [
+    "The bin width in above figure varies for each \"image\" (`smaller`, `larger` and `minicircle`). This is because each\n",
+    "images data is plotted separately (but overlaid on the same graph) and determined dynamically from the range of the data\n",
+    "and is a known shortcoming of Pandas (see [ENH: groupby.hist bins don't match\n",
+    "#22222](https://github.com/pandas-dev/pandas/issues/22222). To get around this you can specify the number of `bins`\n",
+    "explicitly based on the range of _all_ observed data (i.e. `min` to `max`) using `np.linspace()` (from the NumPy\n",
+    "package) along with the number of bins across the _total_ space (bins)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9820d259-680e-49fd-a1eb-e14d200a0b7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "min = df_three_images[\"contour_lengths\"].min()\n",
+    "max = df_three_images[\"contour_lengths\"].max()\n",
+    "bins = 20\n",
+    "df_three_images[\"contour_lengths\"].groupby(\"image\").plot.hist(\n",
+    "    figsize=(16, 9),\n",
+    "    # bins=20,\n",
```

Comment:
Is it intentional that `bins=20` appears twice, with the second occurence commented out?